tests/test_optimizer_dryruns.py

```diff
@@ -677,15 +679,25 @@ def _test_optimize_speed(resources: sky.Resources):
 
 
 def test_optimize_speed(enable_all_clouds, monkeypatch):
-    _test_optimize_speed(sky.Resources(cpus=4))
+    _test_optimize_speed(_make_resources(monkeypatch, cpus='4+'))
```

Comment:
We are using `enable_all_clouds` in this test, which should have similar functionality as `_make_resources`. Should we either remove that fixture from the argument, or add the kubernetes related patch to `enable_all_clouds` as well?

Reply:
Ah good catch, thanks! I've added the patching to `enable_all_clouds` and reverted the changes here.